### PR TITLE
update changelog approver and reviewers for v1.27

### PR DESCRIPTION
**Pull request by harshanarayana**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
Update CHANGELOG reviewers and approver list for `v1.27`

#### Which issue(s) this PR fixes:
None

#### Special notes for your reviewer:
None

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```docs
NONE
```

### CODE REVIEW

File: CHANGELOG/OWNERS [hunk 1]
```diff
@@ -9,14 +9,15 @@ approvers:
   - AuraSinis # 1.24 Release Notes Lead
   - cici37 # 1.23 Release Notes Lead
```
Comment (harshanarayana) [inline CHANGELOG/OWNERS:10]: @JamesLaverack @salaxander @palnabarun Do we need to cleanup any entry from this list or just adding the new one is fine ?

**Comment by harshanarayana** [comment]
/hold 

Wait for https://github.com/kubernetes/org/pull/3936

File: CHANGELOG/OWNERS (hunk 1 above)
Comment (palnabarun) [reply CHANGELOG/OWNERS:10]: For now, it's fine.

**Review by palnabarun — APPROVED** [review]
/lgtm

**Pull request merged**